docs/docs/exit-codes.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Exit Codes 
+---
+
+# Exit Codes 
+
+Marathon follows the [Let-it-crash](https://www.reactivedesignpatterns.com/patterns/let-it-crash.html) pattern. Instead
+of trying to fix an illegal state it will stop itself to be restarted by its supervisor. The following exit codes should
+help you figure out why the Marathon process stopped.
+
+| Exit Code | Reason                                                               |
+|-----------|----------------------------------------------------------------------|
+|100        | `ZookeeperConnectionFailure` - Could not connect to Zookeeper        |
+|101        | `ZookeeperConnectionLost` - Lost connect to Zookeeper                |
+|102        | `PluginInitializationFailure` - Could not load plugin                |
+|103        | `LeadershipLoss` - Lost leadership                                   |
+|104        | `LeadershipEndedFaulty` - Leadership ended with an error             |
+|105        | `LeadershipEndedGracefully` - Leadership ended without an error      |
+|106        | `MesosSchedulerErro` - The Mesos scheduler driver had an error       |
```

Comment:
A missing `r` at the end of `MesosSchedulerErro`.

Reply:
Thanks!